src/test/resources/xml/test.xml

```diff
@@ -0,0 +1,143 @@
+<record xmlns="http://www.loc.gov/MARC21/slim">
+
+    <leader>02286nam a22001810 4500</leader>
+
+    <controlfield tag="001">99123222669705763</controlfield>
+
+    <controlfield tag="005">20191216171630.0</controlfield>
+
+    <controlfield tag="008">000000s2019 dk ||||e 00| 0 dan d</controlfield>
+
```

Comment:
Are you sure we are allowed to share this on the open web?

Reply:
Given that it contains NO info not available on <https://soeg.kb.dk/discovery/fulldisplay?docid=alma99123222669705763&context=L&vid=45KBDK_KGL:KGL&lang=da&search_scope=MyInst_and_CI&adaptor=Local%20Search%20Engine&tab=Everything&query=any,contains,st%C3%B8tte%20til%20et%20liv%20med%20demens.&offset=0> I fail to see the major problem here